cluster-http/src/main/resources/reference.conf

```diff
@@ -10,13 +10,20 @@ akka.management {
   # registers http management to be included in akka-management's http endpoint
   http.route-providers += "akka.management.cluster.ClusterHttpManagementRouteProvider"
 
+  # includes HealthCheckRoutes by default, can be disabled by setting
+  # akka.management.cluster.health-check.default.route-provider = null
```

Comment:
I'm not sure that works, because variable substitution `$` isn't across reference and application conf

I'm also skeptical to that enabling `akka.management.HealthCheckRoutes` belongs to cluster-http.

If we really want to enable `HealthCheckRoutes` by default I think it's better to do that in akka-management, and have another very explicit flag there to disable `akka.management.HealthCheckRoutes`.

Reply:
> I'm not sure that works, because variable substitution `$` isn't across reference and application conf 

I didn't know about that. There are a few things in Typesafe config that surprises me from time to time. That's probably one of them. 

> I'm also skeptical to that enabling `akka.management.HealthCheckRoutes` belongs to cluster-http.

I added there because it's also including the configuration for it. 
https://github.com/akka/akka-management/blob/master/cluster-http/src/main/resources/reference.conf#L13-L19

But I agree, that this should be where `HealthCheckRoutes`. Including its configuration.